In [1]:
print('*** AGA0521 - Manobras Orbitais ***\n')
print('Atividade 6a\n')
print('Aluno: Lyliana Myllena Santos de Sousa')

*** AGA0521 - Manobras Orbitais ***

Atividade 6a

Aluno: Lyliana Myllena Santos de Sousa


Escreva um programa que resolva o problema do Eixo de Torque Livre e faça o teste com os dados do exemplo apresentado na Aula 6, explicitando todos os resultados intermediários e o resultado final.

Iniciaremos definindo os parâmetros que utilizaremos para obter os momentos de inércia:
- I_bx =  1/4 *m_b*r_b^2 + 1/12 * m_b*l_b^2
- I_by = I_bx
- I_bz = 1/2 *m_b*r_b^2

In [2]:
r_b = 0.5 #em m, raio do cilindro
l_b = 1.0 #em m, altura do cilindro
m_b = 300 #em kg, peso do cilindro
l_r = 2.0 #em m, tamanho da haste
m_r = 30 #em kg, peso da haste

#Os momentos de inércia valem:

i_bx = 1/4 *m_b*r_b**2 + 1/12 * m_b*l_b**2
print('i_bx = ',i_bx,'kg*m^2')
i_by = i_bx
print('i_by = ',i_by,'kg*m^2')
i_bz = 1/2 *m_b*r_b**2
print('i_bz = ',i_bz,'kg*m^2')

i_bx =  43.75 kg*m^2
i_by =  43.75 kg*m^2
i_bz =  37.5 kg*m^2


Para o haste delgada R, temos:
- i_rx = 1/12*m_r*l_r**2
- i_ry = 0
- i_rz = i_rx

In [3]:
i_rx = 1/12*m_r*l_r**2
print('i_rx = ',i_rx,'kg*m^2')
i_ry = 0


print('i_ry = ',i_ry,'kg*m^2')
i_rz = i_rx
print('i_rz = ',i_rz,'kg*m^2')

i_rx =  10.0 kg*m^2
i_ry =  0 kg*m^2
i_rz =  10.0 kg*m^2


O momento de inércia da sonda é igual à soma dos momentos de inércia das partes:

In [4]:
i_x = i_bx + i_rx
print('i_x = ',i_x,'kg*m^2')
i_y = i_by + i_ry
print('i_y = ',i_y,'kg*m^2')
i_z = i_bz + i_rz
print('i_z = ',i_z,'kg*m^2')

i_x =  53.75 kg*m^2
i_y =  43.75 kg*m^2
i_z =  47.5 kg*m^2


Pode-se provar que a rotação ao longo do eixo que apresentar o maior momento de inércia é a mais estável. 
Se não houver dissipação da energia de rotação, a rotação será instável para o eixo que apresentar momento de inércia intermediário (entre o maior e o menor).
Se houver dissipação da energia de rotação, a rotação será instável para o eixo que apresentar menor momento de inércia.
Dito isso, temos que:

- O eixo z é o eixo instável com 47.5 kg.m².
- O eixo x, assumindo que não há dissipação de energia, é o eixo estável com 53.75 kg.m².

In [5]:
print('*** AGA0521 - Manobras Orbitais ***\n')
print('Atividade 6b\n')
print('Aluno: Lyliana Myllena Santos de Sousa')

*** AGA0521 - Manobras Orbitais ***

Atividade 6b

Aluno: Lyliana Myllena Santos de Sousa


Utilize o programa desenvolvido na atividade anterior para identificar o eixo (x, y, z ) em torno do qual pode ocorrer a rotação de torque livre estável para um satélite modelado conforme a figura.




In [6]:
r_b2 = 0.75 #em m, raio do cilindro
l_b2 = 1.0 #em m, altura do cilindro
m_b2 = 400 #em kg, peso do cilindro
l_r2 = 4.0 #em m, tamanho da haste
m_r2 = 60 #em kg, peso da haste

#Os momentos de inércia valem:

i_bx2 = 1/4 *m_b2*r_b2**2 + 1/12 * m_b2*l_b2**2
print('i_bx = ',i_bx2,'kg*m^2')
i_by2 = i_bx2
print('i_by = ',i_by2,'kg*m^2')
i_bz2 = 1/2 *m_b2*r_b2**2
print('i_bz = ',i_bz2,'kg*m^2')

i_bx =  89.58333333333333 kg*m^2
i_by =  89.58333333333333 kg*m^2
i_bz =  112.5 kg*m^2


In [7]:
i_rx2 = 1/12*m_r2*l_r2**2
print('i_rx = ',i_rx2,'kg*m^2')
i_ry2 = 0
print('i_ry = ',i_ry2,'kg*m^2')
i_rz2 = i_rx2
print('i_rz = ',i_rz2,'kg*m^2')

i_rx =  80.0 kg*m^2
i_ry =  0 kg*m^2
i_rz =  80.0 kg*m^2


In [8]:
i_x2 = i_bx2 + i_rx2
print('i_x = ',i_x2,'kg*m^2')
i_y2 = i_by2 + i_ry2
print('i_y = ',i_y2,'kg*m^2')
i_z2 = i_bz2 + i_rz2
print('i_z = ',i_z2,'kg*m^2')

i_x =  169.58333333333331 kg*m^2
i_y =  89.58333333333333 kg*m^2
i_z =  192.5 kg*m^2


Pode-se provar que a rotação ao longo do eixo que apresentar o maior momento de inércia é a mais estável. 
Se não houver dissipação da energia de rotação, a rotação será instável para o eixo que apresentar momento de inércia intermediário (entre o maior e o menor).
Se houver dissipação da energia de rotação, a rotação será instável para o eixo que apresentar menor momento de inércia.
Dito isso, temos que:

- O eixo x é o eixo instável com 169.58333333333331 kg.m².
- O eixo z, assumindo que não há dissipação de energia, é o eixo estável com 192.5 kg.m².